crypto/x509/v3_conf.c

```diff
@@ -435,20 +435,48 @@ static X509V3_CONF_METHOD nconf_method = {
     NULL
 };
 
-void X509V3_set_nconf(X509V3_CTX *ctx, CONF *conf)
+int X509V3_set_nconf(X509V3_CTX *ctx, CONF *conf)
```

Comment:
Again not sure if this minor API break is worth it when the only reason for that is adding ctx==NULL test.

Reply:
Reverted